src/main/java/com/treasure/hunt/utils/JavaFxUtils.java

```diff
@@ -1,12 +1,160 @@
 package com.treasure.hunt.utils;
 
+import com.treasure.hunt.analysis.SeriesStatisticObject;
+import javafx.collections.FXCollections;
+import javafx.embed.swing.SwingFXUtils;
+import javafx.geometry.Insets;
+import javafx.scene.control.*;
+import javafx.scene.image.WritableImage;
+import javafx.scene.layout.GridPane;
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+import lombok.extern.slf4j.Slf4j;
+import org.javatuples.Septet;
```

Comment:
use a custom Object, this is not python XD, just make an Object give the stuff names and Voila, use @Value annotation from loombok and u only need to write types and name